src/ResourceManager/Sql/Commands.Sql/Failover Group/Cmdlet/AddAzureSqlDatabaseToFailoverGroup.cs

```diff
@@ -49,13 +49,6 @@ public class AddAzureSqlDatabaseToFailoverGroup : AzureSqlFailoverGroupCmdletBas
         public List<AzureSqlDatabaseModel> Database { get; set; }
 
         /// <summary>
-        /// Gets or sets the tags associated with the Azure SQL Database Failover Group
```

Comment:
Hi Abhijeet. Our team changed the server api need in this cycle and deprecated the need for tags as well as renaming GracePeriodWithDataLossHour to GracePeriodWithDataLossHours. This is not a breaking change because we have not turned on the feature in sql yet and no customer can access this feature at this point. But we are targeting the release next month so I am asked to make this improvement